packaging/amazon-linux-ami/ecs-init.spec

```diff
@@ -152,6 +152,10 @@ if [ -e %{running_semaphore} ]; then
 fi
 
 %changelog
+* Fri May 04 2018 Haikuo Liu <haikuo@amazon.com> - 1.17.4-1
+- Cache Agent vesion 1.17.4
+- Download agent based on the partition
+- Mount Docker plugin files dir
```

Comment:
I don't think you need the last two entries.

Reply:
This is something we have done in the past for clarity. There is no other changelog communicated.